src/app/components/Search/index.tsx

```diff
@@ -90,10 +90,10 @@ export const SearchButton = styled(Button, {
       }
     : {}),
 }))
-SearchButton.defaultProps = {
-  variant: 'contained',
-  color: 'primary',
-}
+
+export const SearchButton = (props: ButtonProps & SearchButtonProps) => (
+  <StyledSearchButton variant="contained" color="primary" {...props} />
+)
```

Comment:
I would rather see something like this:

```suggestion
const PlainTextButton = ({variant = 'text', color: 'inherit', ...restProps}: ButtonProps) => (
  <StyledPlainTextButton variant={variant} color={color} {...restProps} />
)
```

But not a big deal.

Reply:
I vote for original

Reply:
some hope for a nicer solution in the future https://github.com/emotion-js/emotion/issues/2573#issuecomment-1925000208